rcl/include/rcl/time.h

```diff
@@ -431,7 +431,7 @@ rcl_get_time_point_now(rcl_time_point_t * time_point);
 RCL_PUBLIC
 RCL_WARN_UNUSED
 rcl_ret_t
-rcl_enable_ros_time_override(rcl_time_source_t * time_source);
+rcl_ros_time_enable_override(rcl_time_source_t * time_source);
```

Comment:
To be clearer, I would imagine the C++ version of this to be `rcl::enable_ros_time_override()` rather than `rcl::ros_time_source_t::enable_override()`.